anchore_manager/util/logging.py

```diff
@@ -68,7 +73,9 @@ def log_config(config: dict):
         if config['debug']:
             log_level = 'DEBUG'
 
-        logger.set_log_level(log_level, log_to_stdout=True)
+        enable_json_logging = os.getenv('ANCHORE_JSON_LOGGING_ENABLED', '') == 'true'
```

Comment:
Is this for handling bootstrap prior to log setup? Generally, the use of env vars is for the substitution in the config file not directly from the environment in case it conflicts with the config file.

Reply:
Yes it was for the bootstrap prior!